library/src/scala/internal/quoted/ScopeException.scala

```diff
@@ -1,4 +1,4 @@
-package scala.quoted
+package scala.internal.quoted
```

Comment:
What about `private`? Or use `scala.quoted.internal`?

Reply:
There is one use in the compiler. I will try to remove it and make it private.

Reply:
I made it private